packages/common/rollup-configs.js

```diff
@@ -29,20 +31,21 @@ const getBanner = (pkg) => `/*
 /**
  * @returns {import('rollup').RollupOptions}
  */
-export const tsLibConfig = (pkg) => ({
-  input: 'src/index.ts',
+export const tsLibConfig = (pkg, inputFile) => ({
+  input: inputFile,
   output: {
     file: 'dist/index.js',
-    format: 'esm',
     banner: getBanner(pkg),
   },
   plugins: [
     nodeResolve(),
     typescript({
       tsconfig: './tsconfig.json',
+      include: ['src/**/*', '../common/src/**/*'],
```

Comment:
This is equivalent to `tsconfig.json` option
```json
"include": ["src", "../common/src"]
```

but without the above setting, `@rollup/plugin-typescript` will fail to process the code in `common` package as TypeScript code.

The reason for that seems to be

> Source files in these fields are ignored as Rollup's configuration is used instead.

https://github.com/rollup/plugins/tree/master/packages/typescript/#files-include-exclude